universal-application-tool-0.0.1/app/auth/UatProfileData.java

```diff
@@ -18,19 +16,10 @@
  * <p>It is wrapped by UatProfile, which is what we should use server-side.
  */
 public class UatProfileData extends CommonProfile {
-  private Clock clock;
 
-  public UatProfileData() {
-    this(Clock.system(ZoneId.of("America/Los_Angeles")));
-  }
-
-  public UatProfileData(Clock clock) {
-    super();
-    this.clock = Preconditions.checkNotNull(clock);
-  }
+  public UatProfileData() {}
 
-  public UatProfileData(Clock clock, Long accountId) {
-    this(clock);
+  public UatProfileData(Long accountId) {
```

Comment:
suggestion (blocking): we'll want to either call `this()` (to call the other constructor) or `super()` (since that's all the other constructor does)